lgc/patch/SystemValues.cpp

```diff
@@ -425,7 +425,7 @@ Value *ShaderSystemValues::getVertexBufTablePtr() {
   if (!m_vbTablePtr) {
     // Find the node.
     auto vbTableNode = findResourceNodeByType(ResourceNodeType::IndirectUserDataVaPtr);
-    if (vbTableNode) {
+    if (vbTableNode || m_shaderStage == ShaderStageFetch) {
```

Comment:
This conditional here is strange. We don't actually use vbTableNode in the code below.

I suspect we can (and should) change the code to instead just `assert(m_shaderStage == ShaderStageFetch || findResourceNodeByType(...))`.

Reply:
done